src/programmatic_policy_learning/envs/utils/gym_to_gymnasium.py

```diff
@@ -0,0 +1,71 @@
+"""Adapter for interoperating with legacy Gym envs in Gymnasium code."""
```

Comment:
looks awesome! also tagging @jaraxxus-me to let him know that we will soon add this to `prpl-utils`, which should help with some issues he's been facing